WTWSMS/common/religions/00_religions.txt

```diff
@@ -1527,6 +1540,7 @@ muslim = {
 	}
 	druze = {
 		graphical_culture = muslimgfx
+		has_coa_on_barony_only = yes
```

Comment:
Druze are not mainstream kebabs. They have very unique beliefs and symbols that don't coincide with Islam. Why should they be using Muslim symbols of that isn't really a thing in their faith?